wolfcrypt/src/ecc.c

```diff
@@ -4009,8 +4020,8 @@ static int wc_ecc_shared_secret_gen_sync(ecc_key* private_key, ecc_point* point,
 #if defined(WOLFSSL_SP_MATH)
     {
         err = WC_KEY_SIZE_E;
-
         (void)curve;
+        goto errout;
```

Comment:
goto alert.

Reply:
another one of those incumbent (2014) routines that needed a smallstack retrofit, where `goto` was the only way to avert a blowout in the delta.  can't justify the risk+expense of refactoring to avoid the `goto`.

notice that in all recent code, we only use `goto` in retrofits of incumbent code with new flow control requirements, and only to `out` labels for new cleanup semantics.  it's still `goto` but it's the least infractional form of it.